src/robot2017/src/drive_node.cpp

```diff
@@ -74,9 +115,8 @@ int main(int argc, char **argv)
             pub_fb.publish(motorFb);
         }
 
+        publishStatus();
```

Comment:
It doesn't look you're actually setting the values for the status message anywhere before publishing